src/SprykerEco/Zed/Sevensenders/Business/Helper/TokenHelperInterface.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace SprykerEco\Zed\Sevensenders\Business\Helper;
+
+use Generated\Shared\Transfer\SevensendersTokenTransfer;
+
+interface TokenHelperInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\SevensendersTokenTransfer $tokenTransfer
+     *
+     * @return bool
+     */
+    public function isTokenValid(SevensendersTokenTransfer $tokenTransfer): bool;
+}
```

Comment:
please run code sniffer and setup your IDE properly: The newline missing warnings should not be here.